lib/src/test/java/org/openapitools/openapistylevalidator/OpenApiSpecStyleValidatorTest.java

```diff
@@ -0,0 +1,68 @@
+package org.openapitools.openapistylevalidator;
+
+import static org.junit.Assert.assertTrue;
+
+import org.eclipse.microprofile.openapi.OASFactory;
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.junit.Test;
+import org.openapitools.openapistylevalidator.styleerror.StyleError;
+
+import java.util.List;
+
+
+public class OpenApiSpecStyleValidatorTest {
+
+    @Test
+    public void validatePingOpenAPI() {
+        OpenAPI openAPI = createPingOpenAPI();
+        OpenApiSpecStyleValidator validator = new OpenApiSpecStyleValidator(openAPI);
+
+        ValidatorParameters parameters = new ValidatorParameters();
+        List<StyleError> errors = validator.validate(parameters);
+
+        assertTrue(errors.isEmpty());
+    }
+
+    public static OpenAPI createPingOpenAPI() {
+        return OASFactory.createOpenAPI()
```

Comment:
C'est en préparation pour plus tard pour faire de tests j'imagine?

Reply:
@jmini

Reply:
C'est utilisé dans le test `validatePingOpenAPI()` dans ce même fichier.

Reply:
Doh! I didn't realize I was writing in french AND that I missed that! Thanks